app.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+
+const app = express();
+
+const {readFile} = require('fs/promises');
+
+const getTopics = require('./controllers/topics.controllers.js');
+
+app.use(express.json())
```

Comment:
This should be removed as currently, we aren't dealing with any endpoints that require us to be dealing with anything sent in on the request body. When it comes to POST/PATCH/PULL requests - then this will come in handy :)